frontend/package.json

```diff
@@ -8,7 +8,7 @@
     "dist-trees": "ng build --configuration=trees",
     "dist-prod": "ng build --prod --configuration=prod",
     "docs": "yarn run typedoc --out ./src/assets/typedoc --exclude '**/*.spec.ts' ./src/ --module commonjs",
-    "e2e:ci": "ng e2e --port=4200",
+    "e2e:ci": "ng e2e --port=4200 --protractor-config=./development-configurations/protractor.conf.js ",
```

Comment:
this is now the same as the regular `e2e`, can we eliminate `e2e:ci` or is it worth keeping just in case they diverge again in the future?